README.md

```diff
@@ -519,6 +519,30 @@ func isError(error: Error?) -> Bool {
 }
 ```
 
+If there's a case where you want to enter a scope if AT LEAST ONE item is non-nil then use the `??` operator.
+
+```swift
+if let _ = item ?? item1 {
+	// At least one of these is not nil
+}
+```
+
+If you want to enter a scope, only if an object is nil, then you can check directly for nil.
+
+```swift
+if item == nil {
+	// Do stuff..
+}
+```
+
+Additionally, check directly for nil if you are checking for nil AND checking if a bool is YES or NO.
+
+```swift
+if item != nil && propertyBool == true {
```

Comment:
I tested that in playground and I'm getting errors. Plus, that would only work if I want to test if both `testOne` and `testTwo` are `true` or `false`. What if I want to check if `testOne` is `false` and `testTwo` is `true`?